index.html

```diff
@@ -91,6 +91,88 @@ <h2>Statement of Intent</h2>
 
 <section>
 <h2>Code</h2>
+
+	<h3 id="unacceptablebehaviour">
+        <strong>Unacceptable behaviour</strong>
+      </h3>
+
+      <p>Some behaviors run counter to the Code of Ethics and Professional Conduct. This list of unacceptable behaviors does not cover every case. Each person you interact with is unique, and behavior must be assessed on an individual level. Ensuring that your behavior does not have a negative impact is your responsibility.</p>
+
+      <ul>
+        <li>Offensive comments related to gender, gender identity and expression, sexual orientation, disability, mental illness,
+          neurotype, physical appearance, body, age, race, socio-economic status, ethnicity, nationality, language, or religion</li>
+
+        <li>Unwelcome comments regarding a person’s lifestyle choices and practices, including those related to food, health,
+          parenting, drugs, and employment</li>
+
+        <li>Deliberate misgendering or use of ‘dead’ or rejected names</li>
```

Comment:
This is related to safety for transpeople. Misgendering would in this case would be using the incorrect pronouns for a trans or non-binary person. Using someone's old name to refer to them when it no longer applies is harmful in the same way.

Reply:
Thanks for the explanation @AdaRoseCannon. In that case would it be fair to write:

`Deliberately referring to a person using pronouns, proper names or other terms that person has asked not to be used, for example using the wrong gender.`

? I'm just trying to find a form of words that would be super clear to people who might not be familiar with the territory. Also I think it is a plus point that this would catch people who have simply changed their name too, for whatever reason.

Reply:
I would definitely keep the technical term for it in the description somewhere for the people searching for particular items in the code of conduct. Misgendering is also appropriate for those people who have not changed their pronouns but are being bullied by having their gender denied. E.g. Calling a butch woman he or him in order to hurt her.

Perhaps a rewrite like 

`Misgendering someone by deliberately referring to a person using their wrong pronouns. Or by using someone's proper names or other terms that person has asked not to be used, also known as dead-naming.`

Reply:
OK, that's clearer, and a fair point about misgendering people who have not changed their pronouns. 

It's also true that using a name or term that someone has asked not to be used also can apply to people who have not changed their pronouns. This wording looks like it _only_ applies to misgendering, so can we factor out the second sentence into a separate bullet? Forgive my ignorance - I'm not sure if "dead-naming" is a term that is solely used in a misgendering situation.

I'm thinking of the kind of situation in which a person takes another person's surname to associate themselves with someone else (a partner, usually), or perhaps more seriously, stops using a surname to dis-associate themselves from someone (a parent or ex-partner, say) due to that person's bad behaviour.

Reply:
Yes you are correct in that it can be used in in ways which are not related to misgendering although the specific example of deadnaming a transperson is incredibly harmful in the same way misgendering is.

Refusing to use someone's new surname after a relationship change is disrespectful and certainly a microaggression but I think it is less likely to be a psychological trigger for that person.

Reply:
I'm all for avoiding psychological triggering, and don't mean to understate the importance.

Here, I'm advocating that we set the bar much lower than this kind of harm avoidance: if someone has asked for a term not to be used, then it's disrespectful to continue to use it, regardless of the reason why they made the request.

Reply:
+1 to @adarosecannon's proposed revision

Reply:
Thank you. I made the change